src/main/java/nextstep/courses/domain/CourseBuilder.java

```diff
@@ -0,0 +1,68 @@
+package nextstep.courses.domain;
+
+import java.time.LocalDateTime;
+
+public class CourseBuilder{
```

Comment:
```suggestion
public class CourseBuilder {
```
이 클래스에 space convention 위반한 코드가 여러 곳 있음